src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMilliseconds) {
   newCommentTimestampMilliseconds = timeMilliseconds;
 }
 
+/**
+ * Returns an array of the {@code DiscussionComment}s within
+ * {@code TIME_TOLERANCE_MILLISECONDS} milliseconds to the {@code
+ * timestampMilliseconds}. This returns an empty array if no elements are
```

Comment:
nit: Can `This returns an empty ...` be moved to its own paragraph

Reply:
Done